main.cpp

```diff
@@ -0,0 +1,97 @@
+//Coded by Arsen Stoian
+#include <iostream>
+#include <time.h>
+#include "generators.h"
+#include "user_interface.h"
```

Comment:
For readability, local files are included first, in a separate section before library includes.